### ISSUE TITLE
The calculated loss is negative. How to judge its convergence?

### ISSUE THREAD

**Issue by shawroad** [UKPLab/pytorch-bertflow#1]
The calculated loss is negative. How to judge its convergence?

**Comment by kwang2049** [comment]
Hi shawroad,

Yeah, the loss would be negative, which is also the case for the original TF repo. 
According to my experiments, I found the loss will usually be finally saturated around -0.7~-0.4.
So my suggestion is, one can set a fixed number of training steps, e.g. 20K (with batch size 8), etc. Training for 1 epoch on a large corpus is also a good choice.

**Comment by shawroad** [comment]
Thank you for your reply
